.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can we add the `.vscode` path to the `.gitignore` file?

Reply:
I merged `upstream/master` with my `local/master` and then merged that with this branch. I checked the `gitignore` to add `.vscode` and it seems it was added!